api.js

```diff
@@ -174,6 +174,10 @@ var Api = {
             urlSuffix += `&stripItags=1`;
           }
           break;
+        case 'translations':
```

Comment:
I think we should expose the full text v3 api and allow parameters to be passed to specify what to return as opposed to only allowing translations.